src/patternfly/components/Card/card--hidden-input.hbs

```diff
@@ -0,0 +1,6 @@
+{{#> radio radio--modifier="pf-m-standalone"}}
+  {{#> radio-input radio-input--modifier="pf-c-card__hidden-input pf-c-card__input pf-screen-reader" radio-input--attribute=(concat 'id="' card--id '-sr-only-radio" name="' card--id '-sr-only-radio" aria-labelledby="' card--id '" ' card--hidden-input--IsDisabled )}}{{/radio-input}}
```

Comment:
Is it necessary to have this as `.pf-c-radio__input.pf-c-card__input.pf-c-card__hidden-input`? We've avoided using multiple component classes between components on a single element, and have backed out the handful of places where we have done that, including some in this v5 release. For example, separating `.pf-c-skip-to-content` and `.pf-c-button` - https://github.com/patternfly/patternfly/issues/4972

Reply:
Actually just poking around, is `.pf-c-card__hidden-input` relevant/needed? doesn't seem to do anything and isn't documented.

Reply:
@mcoker  To clarify, are you suggesting changing from:
```
    div.pf-c-card__actions
      div.pf-c-radio.pf-m-standalone
          input.pf-c-radio__input.pf-c-card__input.pf-screen-reader
          label.pf-c-radio__label.pf-c-card__label
```
to something like:
```
    div.pf-c-card__actions
      div.pf-c-radio.pf-m-standalone
          div.pf-c-card__input
            input.pf-c-radio__input.pf-screen-reader
          div.pf-c-card__label
            label.pf-c-radio__label
```

Reply:
Ok - before you've been reluctant to select one component from another app I wasn't sure what you meant.